contracts/eosio.system/src/wps.cpp

```diff
@@ -161,7 +161,7 @@ namespace eosiosystem {
         eosio::action(
                 eosio::permission_level{get_self() , "active"_n },
                 "eosio.token"_n, "transfer"_n,
-                std::make_tuple( get_self(), account, transfer_amount, std::string("Your worker proposal has been approved."))
+                std::make_tuple( "eosio.saving"_n, account, transfer_amount, std::string("Your worker proposal has been approved."))
```

Comment:
for consistency with the previous suggestion (using the `saving_account` global): `std::make_tuple( saving_account, account, transfer_amount, std::string("Your worker proposal has been approved."))`